docs/conf.py

```diff
@@ -216,4 +216,8 @@
 ]
 
 
+# -- Option to allow errors in jupyter notebooks --------------------------
 
+# TODO: this is temporary as long as the demo notebooks are not fully
+# functional. It should be removed when all errors have been resolved.
+nbsphinx_allow_errors = True
```

Comment:
I just found that this can be configured on a per-notebook basis, in the JSON metadata of the notebook:

```
"nbsphinx": {
  "allow_errors": true
},
```

See https://nbsphinx.readthedocs.io/en/0.2.15/allow-errors.html.

Reply:
:+1: Awesome